en-US/Translation.xml

```diff
@@ -78,8 +78,9 @@
         It is important to provide sufficient spacing between elements in your documents to facilitate reading and comprehension.
         You can include a lot of information in a few short paragraphs but readers need to be able to process that information in chunks.
         The same consideration applies to lists.
-        If you use DocBook to build itemized, ordered, or variable (definition) lists, you can use the <option>compact</option> or <option>normal</option> attributes to specify the spacing between list items.
-        DocBook uses the <option>normal</option> spacing attribute by default.
+        If you use DocBook or AsciiDoc to build itemized or ordered lists, you can choose between normal spacing or the <option>compact</option> attribute to specify the spacing between list items.
+		In AsciiDoc, compact spacing is not supported for variable (definition) lists.
+        DocBook and AsciiDoc use the <option>normal</option> spacing attribute by default.
```

Comment:
We're in the process of removing language-specific guidelines from the style guide. We shouldn't be adding more. People who use the style guide might write in anything from LaTex to googledoc to Word or even Lotus Notes <gulp>.

Reply:
Fixed